StackTraceParser.csproj

```diff
@@ -1,64 +1,78 @@
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <LangVersion>5</LangVersion>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <TargetFramework>net452</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <NoWarn>$(NoWarn);NU5128;NU5105</NoWarn>
   </PropertyGroup>
+
   <ItemGroup>
-    <Reference Include="nunit.framework, Version=2.6.4.14350, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
-      <HintPath>packages\NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit" Version="3.12.0" PrivateAssets="All" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Public.cs" />
-    <Compile Include="StackTraceParser.cs" />
-    <Compile Include="StackTraceParserTests.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Properties\" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>
-    </PostBuildEvent>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>StackTraceParser.Source</PackageId>
+    <Title>StackTraceParser (C# Source)</Title>
+    <PackageTags>stack;trace;parser;diagnostics;exception</PackageTags>
+    <Description>Parser for .NET and Mono stack traces.</Description>
+    <Copyright>Copyright &#169; 2020 Atif Aziz. All rights reserved.</Copyright>
+    <PackageVersion>1.3.1</PackageVersion>
+    <Authors>Atif Aziz</Authors>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <DefaultLanguage>en-us</DefaultLanguage>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageOutputPath>.\dist</PackageOutputPath>
+    <PackageReleaseNotes>See https://github.com/atifaziz/StackTraceParser/milestone/3</PackageReleaseNotes>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
+
+  <UsingTask
+	TaskName="ReplaceFileText"
```

Comment:
Can we replace the tabs with spaces since this sections seems to be misaligned with the rest and therefore harder to parse. Otherwise brilliant and thanks for taking the trouble to get this in so we can match how it was distributed so far.

This is an old project and I ought to add an `.editorconfig` to avoid such inconsistencies from being introduced (accidentally).

Reply:
Yeah editorconfig would have helped. I can try to fix it no worries